test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -187,9 +187,14 @@ public void Func([Queue(QueueName)] out string x)
         public void Fails_When_No_Storage_is_set()
         {
             var host = TestHelpers.NewJobHost<ProgramSimple>();  // no storage account!
-
+            string message = "Microsoft Azure WebJobs SDK Storage connection string is missing or empty. " +
```

Comment:
Consider just calling the FormatParseAccountErrorMessage method to get this string, rather than duplicate it here.